addons/stock/models/stock_quant.py

```diff
@@ -737,7 +737,7 @@ def _get_removal_strategy_sort_key(self, removal_strategy):
         if removal_strategy == 'lifo':
             reverse = True
         elif removal_strategy == 'closest':
-            key = lambda q: (q.location_id, -q.id)
+            key = lambda q: (q.location_id.complete_name, -q.id)
```

Comment:
could you write a small test to ensure the complete name is used instead of the id ?